github/pulls_test.go

```diff
@@ -232,31 +233,63 @@ func TestPullRequestsService_Create_invalidOwner(t *testing.T) {
 }
 
 func TestPullRequestsService_Edit(t *testing.T) {
-	setup()
-	defer teardown()
+	tests := []struct {
+		input        *PullRequest
+		sendResponse string
+
+		wantUpdate   *pullRequestUpdate
```

Comment:
I see.

Yeah, `encoding/json` encoder uses the order of fields in a struct when encoding, so it is deterministic. It even sorts map keys.